css/styles.css

```diff
@@ -0,0 +1,86 @@
+html {
+  margin: 0;
+  padding: 0;
+}
+
+body, p {
+  margin: 0;
+  padding: 0;
+  font-family: -apple-system, BlinkMacSystemFont, "helvetica neue", helvetica, arial, sans-serif;
+  font-size: 18px;
+  line-height: 24px;
+  letter-spacing: .1em;
+  background: white;
+  text-transform: lowercase;
+}
+
+.homebox {
+  padding-top: 2em;
+  padding-left: 2em;
+  width: 50%;
+  /* border: 1px red solid; */
+}
+
+.textbox {
+  padding-top: 1em;
+  margin: 0 auto;
+  height: auto;
+  width: 50%;
+}
+
+.a {
```

Comment:
lets not call these `a` and `b` -- it's too similar to the anchor element's name + confusing imo.. could we call em left and right or something instead?

Reply:
totally, let's do that. so left and right and then something like leftcredit and rightcredit to indicate the names of the people in the conversation. im proposing that we dont repeat the names with every line but just have them as a key, once on the screen... whatcha think?

Reply:
that makes sense to me about the names -- can do whatever if you show me how it needs to be laid out

Reply:
yup! take a look at the html pages when u get a sec